sources/Metal/Renderers/iTermCursorGuideRenderer.m

```diff
@@ -1,60 +1,96 @@
 #import "iTermCursorGuideRenderer.h"
 
 @interface iTermCursorGuideRendererTransientState()
-@property (nonatomic, strong) id<MTLTexture> texture;
+@property (nonatomic, strong) id<MTLTexture> horizontalTexture, verticalTexture;
```

Comment:
Style nit: properties and variable declarations are always one per line. Same on line 7, 13, and 14.

Reply:
Done.